src/Views/IMEView.cpp

```diff
@@ -0,0 +1,162 @@
+#include "IMEView.h"
+
+
+IMEView::IMEView(IMEViewResult *result, const char title[] , const char showText[]) :
+	IMEView(result, title, showText, nullptr, SCE_IME_DIALOG_MAX_TEXT_LENGTH)
+{}
+
+IMEView::IMEView(IMEViewResult *result, const char title[] , const char showText[],
+                 SceUInt32 maxInputLength) :
+	IMEView(result, title, showText, nullptr, maxInputLength)
+{}
+
+
+IMEView::IMEView(IMEViewResult *result, const char title[] , const char showText[],
+                 const char initialText[]) :
+	IMEView(result, title, showText, initialText, SCE_IME_DIALOG_MAX_TEXT_LENGTH)
+{}
+
+IMEView::IMEView(IMEViewResult *result, const char title[] , const char showText[],
+		const char initialText[], SceUInt32 maxInputLength) {
+	log_printf(DBG_DEBUG, "Created IMEView \"%s\"", title);
+	utf8_to_utf16((uint8_t *)title, _title);
```

Comment:
Have you tried using [C++ converters](https://stackoverflow.com/a/38383389) instead?

Reply:
Actually no. I just copied the conversion code as found in may other homebrews.